src/Pos/Launcher/Runner.hs

```diff
@@ -199,6 +203,7 @@ runRealModeDo discoveryCtx slottingCtx transport np@NodeParams {..} sscnp
                runProduction .
                    runResourceT .
                    usingLoggerName lpRunnerTag .
+                   runJsonLogT' mJLHandle .
```

Comment:
@gromakovsky good point, I think you're right, this is indeed a problem. We never ran into this problem during benchmarking, so I'm sure ekg monitoring doesn't need json logging, so using `runWithoutJsonLogT` sounds like the right solution.